### PR TITLE
Create project

### PR DESCRIPTION
**Pull request by luminitacristinalazarescu**
[@mei-li](https://github.com/mei-li) and [@zampeta](https://github.com/zampeta) I created a new project and a requirements file.

Please review this.

### CODE REVIEW

**Comment by mei-li** [comment]
add a .gitignore file to exclude the .pyc files from the commit https://help.github.com/articles/ignoring-files/
In order to change this commit, you can reset it first, git reset --soft HEAD~ (it will un-do the commit, but keep the changes staged, then add the .gitignore, maybe you will need to unstage the pyc manually, I am not sure.)
And after push with -f (force push)

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+django==1.8
+ipdb
+nose
```
Comment (mei-li) [inline requirements.txt]: nose and ipdb would be better to be included in another requirements file eg. dev.txt as they are not requirements for the live system but for testing/development. But it is totally fine for now

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+*.pyc
```
Comment (mei-li) [inline .gitignore]: add also *.sqlite3, to avoid commiting the DB

Comment (luminitacristinalazarescu) [inline .gitignore]: Done

**Comment by mei-li** [comment]
:+1:

**Pull request merged**